packages/govuk-frontend/src/govuk/components/tabs/tabs.mjs

```diff
@@ -64,6 +73,28 @@ export class Tabs extends GOVUKFrontendComponent {
     this.boundTabKeydown = this.onTabKeydown.bind(this)
     this.boundOnHashChange = this.onHashChange.bind(this)
 
+    const $tabList = this.$module.querySelector('.govuk-tabs__list')
+    const $tabListItems = this.$module.querySelectorAll(
+      'li.govuk-tabs__list-item'
+    )
+
+    if (!$tabList) {
+      throw new ElementError($tabList, {
```

Comment:
Sorry, this old chestnut again 😆

```suggestion
      throw new ElementError(null, {
```

Reply:
Not caused by this PR, but should really match up the JSDoc at the other end

i.e. Adding `| null` to the allowed types

```mjs
/**
 * @param {Element | null} element - The element in error
 */
```

Reply:
Implemented the suggestion and add `| null` to the ElementError JSDoc.